tests/test_inference.py

```diff
@@ -5495,12 +5495,14 @@ def add(x, y):
                 add(*nums)
```

Comment:
We should be able to leave the sense of this test as is without changing the assertions by:

```suggestion
            if nums is not None:
                add(*nums)
```